extras/kms/migrations/postgres/02_version.up.sql

```diff
@@ -0,0 +1,31 @@
+begin;
+
+create table kms_schema_version(
+    version text not null,
+    create_time kms_timestamp,
+    update_time kms_timestamp
+);
+
+-- ensure that it's only ever one row
+create unique index kms_schema_version_one_row
+ON kms_schema_version((version is not null));
+
+ -- define the immutable fields for kms_root_key (all of them)
+create trigger kms_immutable_columns
+before
+update on kms_schema_version
+  for each row execute procedure kms_immutable_columns('create_time');
+
+create trigger kms_default_create_time_column
+before
+insert on kms_schema_version
+  for each row execute procedure kms_default_create_time();
+
+create trigger update_time_column 
+before 
+update on kms_schema_version 
+	for each row execute procedure update_time_column();
+
+insert into kms_schema_version(version) values('v0.0.1');
```

Comment:
So this version will be tied to a tag? I would expect this to correspond to the latest migration number, just curious why the use of this string?

Reply:
A future PR will include a runtime check of this against the tagged module version.  We can't use the "standard" migration numbers since the migrations from this module may likely need to be integrated with an application's existing migrations which will likely have different migration sequences/timestamps.